core/app/helpers/spree/base_helper.rb

```diff
@@ -25,6 +25,24 @@ def display_price(product_or_variant)
         to_html
     end
 
+    def localized_item_link(item, locale)
```

Comment:
@MatthewKennedy I think this can be moved to frontends `NavigationHelper`, also let's prefix it with `spree_`